plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -1512,6 +1512,26 @@ public static void drawMessageOverControl(Control control, GC gc, String message
         }
     }
 
+    /**
+     * Method designed to draw message on top of control with left bottom
+     * positioning
+     * @param control - ui control
+     * @param gc      - graphical container
+     * @param message - custom message text
+     */
+    public static void drawMessageOverControlOnLeftBottom(Control control, GC gc, String message) {
```

Comment:
Let's leave this method in panel editor. UIUtils is already too big and this implementation looks quite specific.